content/software-security/what-are-containers/index.md

```diff
@@ -0,0 +1,64 @@
+<!-- Place a graphic here showing the result of instantiating a container -->
+
+To start creating a container image, first, a *base image* must be selected. A base image is a foundational image that can be built upon through the addition of image *layers*. Typically, base images come pre-bundled with a specific Linux distribution. Every distribution differs in its size, dependencies, and functionality, making certain distributions better suited for certain images over others.
+
+Once a base image has been selected, introducing tailored functionality (such as your application-specific code or dependencies) to the image can be done in a few different ways. Many developers today use [Docker](https://www.docker.com/), a tool used to deploy containerized applications, which ingests a [*Dockerfile*](https://docs.docker.com/engine/reference/builder/), a machine-readable configuration document containing the instructions to assemble an image with multiple layers. Aside from Dockerfiles, tools such as [apko](/open-source/apko/overview/) and [ko](https://github.com/ko-build/ko) can be used to build images, though they produce images using a single-layer construction method. 
+
+For example, say you want to build a container to run your Python application. You can start your container image with a Python base image, such as a [Wolfi-based image](/open-source/wolfi/wolfi-with-dockerfiles/). Then, you can add your application and its relevant dependencies to the image through the layers in a Dockerfile. The resultant image is now ready to be deployed with your application bundled inside.
+
+
+## Instantiating Containers
+
+Once you have selected or assembled a container image for your application, you will use it to instantiate a running container. Deploying live containers from an image requires the use of a *container engine*, which is the software that allows for the hosting of multiple containers on one shared machine. [Docker Engine](https://docs.docker.com/engine/) is a prominent example of a container engine.
+
+A container engine communicates with the kernel of the operating system it is being run on. Within a container engine, multiple containers run independently of each other. These containers are composed of the code, dependencies, and configurations of their parent images. The following graphic depicts this hierarchical relationship between containers and the container engine.
+
+<img src="container_structure.png" alt="Containers run independently of each other on a container engine, the software that communicates with a host operating system kernel. Each container contains the application and dependencies it needs to run." width="400" />
```

Comment:
I could not figure out how to resize an image using markdown - it seemed to automatically try and fit the entire width of the page. but with this HTML solution, I don't know how to center the image! :( looking for help here since I know HTML isn't what is recommended by our readme

Reply:
Did you try adding `class="center"` to the `img` tag? Maybe after the `width` thing

If you did try that and it doesn't work, I"m not sure how to resolve that. Looking at our CSS it seems like all images should center automatically? I think you've also got the wrong image reference here, so I can't see the image in the article

Reply:
Both show up locally for me, but only this HTML tag one shows up on the deploy preview. The other image that I included using markdown isn't showing :( I will try the class attribute now

Reply:
Netlify is a little funny, I think you can use markdown that will render fine on our prod site